apps/mobile/src/components/NetworkBadge/NetworkBadge.tsx

```diff
@@ -0,0 +1,27 @@
+import { Text, Theme, View } from 'tamagui'
+import { Logo } from '@/src/components/Logo'
+import { Chain } from '@safe-global/store/gateway/AUTO_GENERATED/chains'
+
+type Props = {
+  network: Chain
+  themeName?: string
+}
+export const NetworkBadge = ({ network }: Props) => {
```

Comment:
it would be nice to have a storybook for this component

Reply:
done